packages/docusaurus-plugin-content-docs/src/__tests__/index.test.js

```diff
@@ -7,7 +7,7 @@
 
 import '@babel/polyfill';
 import path from 'path';
-import loadSetup from '../../../docusaurus/test/loadSetup';
+import loadSetup from '../../../docusaurus/lib/server/load/__tests__/loadSetup';
```

Comment:
Maybe we'll create @docusaurus/test-utils in the future to easily load up fixtures. Not going to do it in this PR.